guide/Overview

```diff
@@ -0,0 +1,57 @@
+# MESG Overview
+
+MESG is an open-source development framework that combines Service-Oriented and Event-Driven Architectures to enable decentralized application processing and hosting.
+
+It consists of a network of services made interoperable via the Engine, a decentralized technology-agnostic message broker. Services featuring any language can be bought and sold in the open Marketplace and used within any application to accelerate efficiency. 
```

Comment:
Let's replace "bought and sold" by something like "rent" or "pay per execution", "pay per use"